File: 101/exercise_rework/easy_9/counting_up.rb
# counting_up.rb

def neg_sequence(int)
  arr = []
  int.upto(-1) { |int1| arr << int1  }
  arr
end

def sequence(int)
  return neg_sequence(int) if int < 0
  arr = []
  1.upto(int) { |int1| arr << int1 }
  arr
end

File: 210/js_small_problems/easy_5/counting_up.js
// counting_up.js

function sequence(int) {
  var sequentialIntegers = [];

  for (var i = 1; i <= int; i += 1) {
    sequentialIntegers.push(i);
  }
  return sequentialIntegers;
}

console.log(sequence(5));    // [1, 2, 3, 4, 5]
console.log(sequence(3));    // [1, 2, 3]
console.log(sequence(1));    // [1]